SnakeAndLadderGameProject/src/IBlock.java

```diff
@@ -0,0 +1,9 @@
+public class IBlock {
```

Comment:
better name?

Reply:
since it has start and end, should not be named Block

Reply:
yeah changed it.